packages/orchestration/src/exos/chain-hub.js

```diff
@@ -370,7 +377,7 @@ export const makeChainHub = (zone, agoricNames, vowTools) => {
       const connectionInfo = await vowTools.asPromise(
         chainHub.getConnectionInfo(primary, counter),
       );
-      return /** @type {[ActualChainInfo<C1>, ActualChainInfo<C2>, IBCConnectionInfo]} */ ([
+      return /** @type {[ChainInfo, ChainInfo, IBCConnectionInfo]} */ ([
```

Comment:
Thinking about this change. I suppose at some point - when non-cosmos chains are in `fetched-chain-info.js` - we'll want to restore this